stacks/apps/cms.yml

```diff
@@ -137,6 +138,65 @@ Resources:
       ManagedPolicyArns:
         - Fn::ImportValue: !Sub ${SecretsStackName}-SecretsInstanceDecryptPolicyArn
         - Fn::ImportValue: !Sub ${SecretsStackName}-SecretsInstanceAccessPolicyArn
+      Policies:
+        - PolicyDocument:
+            Statement:
+              - Action: sns:Publish
+                Effect: Allow
+                Resource: !Ref PorterJobExecutionSnsTopicArn
+            Version: "2012-10-17"
+          PolicyName: PorterSnsPublish
+        - PolicyDocument:
+            Statement:
+              - Action: sqs:*
+                Effect: Allow
+                Resource:
+                  - !GetAtt AudioCallbackQueue.Arn
+                  - !GetAtt ImageCallbackQueue.Arn
+                  - !GetAtt PodcastImportQueue.Arn
+                  - !GetAtt SearchIndexerQueue.Arn
+                  - !GetAtt DefaultJobQueue.Arn
```

Comment:
Does this cover dead letter queues?  Or I guess ... that's built into SQS, and CMS would never pick up a message from one.  Nevermind!